src/plugins/requestAuthPlugin.js

```diff
@@ -47,22 +54,20 @@ export const requestAuthorizer = auth$ => request => {
 	// always need oauth_token, even if it's an anonymous (pre-reg) token
 	// This is 'deferred' because we don't want to start fetching the token
 	// before we know that it's needed
-	const request$ = Rx.Observable.of(request);
-	const authType = request.state.oauth_token && 'cookie';
-
 	request.log(['info', 'auth'], 'Checking for oauth_token in request');
-	return Rx.Observable.if(
-		() => authType,
-		request$
-			.do(() => request.log(['info', 'auth'], `Request contains auth token (${authType})`)),
-		request$
-			.do(() => request.log(['info', 'auth'], 'Request does not contain auth token'))
-			.flatMap(request =>
-				auth$(request)
-					.do(applyAuthState(request, request.plugins.requestAuth.reply))
-					.map(() => request)
-			)
-	);
+
+	const authType = request.state.MEETUP_MEMBER && 'MEETUP_MEMBER' ||
```

Comment:
You know, I think it may just be a naming confusion for me. Since the cookies are `MEETUP_MEMBER` and `MEETUP_MEMBER_DEV`,  I assume that other usages of `MEETUP_MEMBER` would reference a prod member. Nitpick, but could we just change the state key to be `request.state.MEMBER`?